bigtable/beam/change-streams/pom.xml

```diff
@@ -102,4 +102,4 @@
       <artifactId>gax</artifactId>
     </dependency>
   </dependencies>
-</project>
+</project>
```

Comment:
Nit: consider reverting this change (GitHub flags missing new lines at the end of files). Also present in the `kotlin-ktor` `pom.xml` change. [Background info](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)